<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GPBR Time Clock</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="employeeList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>821</width>
      <height>461</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>181</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>120</pointsize>
     </font>
    </property>
    <property name="text">
     <string>▲</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>250</y>
      <width>181</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>120</pointsize>
     </font>
    </property>
    <property name="text">
     <string>▼</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="dateTimeGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>821</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Current Date and Time</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="timeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>19</y>
       <width>801</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>120</pointsize>
      </font>
     </property>
     <property name="text">
      <string>12:00 PM</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="dateLabel">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>190</y>
       <width>801</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Wednesday, December 25, 2016</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="clockButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>490</y>
      <width>181</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clock
In/Out</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
